server/infra/database/PgRepositories.js

```diff
@@ -32,6 +32,10 @@ class CaptureRepository extends BaseRepository {
         .limit(options.limit)
         .offset(options.offset);
     }
+
+    async add(capture) {

```

Comment:
Ensure the signature expected by the `Repository` in the model is implemented here, even if it is a simple wrapper. This guarantees the code doesn't bypass any implementation in the repository implementations.